### ISSUE TITLE
tox 4 compatibility

### ISSUE THREAD

**Issue by gaborbernat** [signalpillar/tox-battery#31]
This feature is now supported natively with tox 4.

**Comment by signalpillar** [comment]
Oh, nice! Thank you for the update. I haven't tried it yet.

**Comment by gaborbernat** [comment]
It's not out yet and likely will take a few more months maturing to be widely available 😀only so much free time I have...🤷‍♂️

**Comment by signalpillar** [comment]
Yeah, I just checked it's not released yet. I am glad it's moving forward and will be part of core functionality.

**Comment by gaborbernat** [comment]
There's a pre release if you want to check it https://pypi.org/project/tox/4.0.0a6/

**Comment by gaborbernat** [comment]
I am withdrawing this request and closing the issue, as it has had no follow-up in a long time. Thanks for taking a look.
